Gruntfile.js

```diff
@@ -55,13 +55,13 @@ module.exports = function(grunt) {
 
   // enable plugins
   grunt.loadNpmTasks('grunt-contrib-compress');
-  grunt.loadNpmTasks('grunt-contrib-csslint');
+  // grunt.loadNpmTasks('grunt-contrib-csslint');
```

Comment:
remove commented out code here too

Reply:
Why is CSS linting being removed?

Reply:
There is a bug in CSS lint in that it can't handle custom vars. Custom vars is needed if we want to be able to define an aspect ratio for the video or canvas. :/